GTG/gtk/browser/simple_color_selector.py

```diff
@@ -234,12 +234,12 @@ def __build_custom_palette(self):
         buttons_hbox = Gtk.Box()
         cc_vbox.pack_start(buttons_hbox, True, True, 0)
         self.add_button = Gtk.Button()
-        self.add_button.set_label("Add custom color")
+        self.add_button.set_label(_("Add Custom Color..."))
```

Comment:
Unicode typography for ellipsis - https://developer.gnome.org/hig/stable/typography.html.en